### ISSUE TITLE
Look into Facebook Messenger API

### ISSUE THREAD

**Issue by ankitr** [ProjectCopilot/mailroom#14]
[Facebook released a platform for services to automatically use Messenger.](https://developers.facebook.com/docs/messenger-platform) We should look into the viability of this as another contact method.

**Comment by gmittal** [comment]
Ok. Sounds good.

**Comment by ankitr** [comment]
I'll talk with a contact on the FB messenger team to see if we can get access to the beta.

**Comment by gmittal** [comment]
Update: Facebook Pages are really bad.

**Comment by gmittal** [comment]
Status on this @ankitr?

**Comment by ankitr** [comment]
Pages are being a pain to work with. I think we'll hold off for a bit.

**Comment by gmittal** [comment]
GitHub has declared this issue stale.

**Comment by gmittal** [comment]
Are we still doing this?

**Comment by gmittal** [comment]
I think it would be super awesome.

**Comment by ankitr** [comment]
I'm down. How hard would it be to write a module?

**Comment by gmittal** [comment]
No idea.

**Comment by gmittal** [comment]
Seems as though this has gotten easier to do since summer (Long live the bot revolution). May be worth looking into again this year.

**Comment by gmittal** [comment]
It has gotten much easier to build Facebook Messenger bots. However, Facebook has placed many security features to prevent bots from randomly messaging any user. From what I understand, a user first has to interact with a page (by sending a query directly through Messenger) and only then can a page start interacting with that user. Even then, you have to go through special approval from Facebook's Messenger Team if you want to still be able to send messages to that user after 24 hours without them messaging your bot again.

This is getting rather complicated for something that should be a lot simpler to implement. Taking this off of MVP for now.

**Comment by ankitr** [comment]
Makes sense. Let's look at what it takes to get approval from FB's team.
